changelogs/fragments/3875-icinga2-inv-fix.yml

```
@@ -0,0 +1,10 @@
+---
+breaking_changes:
+  - icinga2 - Inventory object names will now default to the host object name by default as opposed to the address field. This can be changed using ``inventory_attr`` in your config file.
```

Please format changelog fragments as follows:
```suggestion
  - icinga2 inventory plugin - inventory object names will now default to the host object name by default as opposed to the address field. This can be changed using ``inventory_attr`` in your config file (https://github.com/ansible-collections/community.general/issues/3875, https://github.com/ansible-collections/community.general/pull/3906).
```